newsletter.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
Let's remove this file from the pull request, as it isn't relevant to what we are doing, maybe even add it into the `.gitignore`?